clang/test/CodeGenSYCL/native_cpu_basic.cpp

```diff
@@ -50,8 +50,8 @@ void gen() {
 }
 
 // Check name mangling 
```

Comment:
Does this test check name mangling specific to native cpu anymore?   If not, can we fix the comments here, at the top and the typeinfo name check below?

Reply:
Yes you are right, it's not specific to native cpu anymore so I've removed the check, thank you